tests/unit/modules/network/ios/fixtures/ios_vlans_config.cfg

```diff
@@ -0,0 +1,29 @@
+VLAN Name                             Status    Ports
+---- -------------------------------- --------- -------------------------------
+1    default                          active    Gi0/1, Gi0/2
+150  VLAN0150                         active
```

Comment:
Can we have a vlan with "remote" in the name to make sure facts doesn't throw a traceback like in the reported bug?